nexus/deployment/src/planner.rs

```diff
@@ -66,6 +80,41 @@ impl<'a> Planner<'a> {
                 continue;
             }
 
+            // Now we've established that the current blueprint _says_ there's
+            // an NTP zone on this system.  But we must wait for it to actually
+            // be there before we can proceed to add anything else.  Otherwise,
+            // we may wind up trying to provision this zone at the same time as
+            // other zones, and Sled Agent will reject requests to provision
+            // other zones before the clock is synchronized.
```

Comment:
I'm not sure this is exactly what we want to check. During RSS we wait not only for the NTP zone to exist, but for time to be synchronized - I assume that's what the subsequent service zones really care about. Should inventory collect the [timesync state](https://github.com/oxidecomputer/omicron/blob/62547d22e74c4bb134b8d65e373ace793de7529f/sled-agent/src/http_entrypoints.rs#L662-L671) of each sled, so here we can wait for [`timesync.sync`](https://github.com/oxidecomputer/omicron/blob/62547d22e74c4bb134b8d65e373ace793de7529f/sled-agent/src/params.rs#L792) instead of the NTP zone's presence?

Reply:
Oh! Right, completely forgot about that.